src/wakeflow/non_linear_perts.py

```diff
@@ -440,6 +479,8 @@ def _get_non_linear_perts(self) -> None:
         print('Propagating inner wake... ')
 
         timer_0 = time.perf_counter()
+        
```

Comment:
Please add a comment.

I am also not the biggest fan of re-using the same variable `edge_limit` and overwriting the old value. I would prefer to have `self.edge_limit_inner` and `self.edge_limit_outer` and to pass through the correct one for each call of `_solve_burgers`.